heed/src/env.rs

```diff
@@ -297,6 +297,11 @@ impl Env {
                 let stat = unsafe { stat.assume_init() };
 
                 size += compute_size(stat);
+
+                // if the db wasn’t already opened
+                if !dbi_open.contains_key(&dbi) {
+                    unsafe { ffi::mdb_dbi_close(self.0.env, dbi) }
+                }
```

Comment:
A better approch would be to not open it a second time if you find it in the `dbi_open` map, right?

Reply:
but how can I know its dbi without opening it?
I looked at the code in `raw_open_dbi` and noticed the first action was to call mdb_dbi_open so I don’t think there is currently a datastructure doing the association name -> dbi 🤔

Reply:
Yeah, you are right, I didn't notice that there was no name associated with the dbi in this map. Yup, it should be good the way you've done it!